dgbpy/dgbscikit.py

```diff
@@ -425,6 +426,12 @@ def unscale( samples, scaler ):
 
   return samples
 
+def isMultiLabel(setup):
+  out_shape = setup[dgbhdf5.outshapedictstr]
+  if isinstance(out_shape, int) and not setup[dgbhdf5.classdictstr]:
```

Comment:
use:
dgbpy.hdf5.isClassification( setup )
instead of:
setup[dgbhdf5.classdictstr]

Reply:
Look more in detail at the API in dgbpy.hdf5. It contains all the tools to parse and retrieve information from such setup dictionary. The entire implementation would be:
if not dgbpy.hdf5.isRegression( setup ) or dgbpy.hdf5.isImg2Img( setup ):
  return False;
return dgbpy.hdf5.getNrOutputs( setup ) > 1

You may even want to move this function to dgbpy.hdf5 and call it isMultiLabelRegression( info )